src/scripts/visualizations/line.js

```diff
@@ -139,7 +139,7 @@
         }
 
         function renderMarkers() {
-            var animationDelay = options.line.marker.animationDelay;
+            var animationDelay = (options.line.marker.animationDelay == null) ? duration : options.line.marker.animationDelay;
```

Comment:
the check is fine, just wanted to make sure jshint was working (and there aren't other jshint errors we're letting through).